QemuPkg/Bin/qemu_ext_dep.yaml

```diff
@@ -0,0 +1,19 @@
+##
+# Downloads the exact version of QEMU that is known to work for the
+# current version of mu_tiano_platforms.
+#
+# Copyright (c) 2019, Microsoft Corporation
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+{
+  "scope": "edk2-build",

```

Comment:
Similar to the other branch, probably worth splitting the scope for windows/linux as the is a pretty good size exttdep.

Reply:
I split qemu into two extdeps and updated the scopes to global-win and global-nix as appropriate